### ISSUE TITLE
pgo scaledown does not show replica name

### ISSUE THREAD

**Issue by GeorgFleig** [CrunchyData/postgres-operator#1247]
**Describe the bug**
```
pgo scaledown -n postgres pg2 --query

Cluster: pg2
REPLICA             	STATUS    	NODE      	REPLICATION LAG
pg2-axxw            	running   	kubernetes-node-1	           0 MB
                    	running   	          	           0 MB
```

Pods:
```
NAME                                        READY   STATUS      RESTARTS   AGE
backrest-backup-pg2-pgs6v                   0/1     Completed   0          127m
pg2-axxw-7f9cd64486-b87np                   1/1     Running     0          126m
pg2-backrest-shared-repo-945d495c9-dxtkf    1/1     Running     0          128m
pg2-bb955878d-pk5dr                         1/1     Running     0          128m
pg2-stanza-create-tm555                     0/1     Completed   0          127m
pg2-xlln-b9ccfc4b8-cxbpn                    1/1     Running     0          126m
```

And the pgo apiserver log shows the pod name correctly:
```
time="2020-01-28 12:35:04 +0000" level=debug msg="patroni instance info: [{pg2-axxw-7f9cd64486-b87np  0 running 1} {pg2-bb955878d-pk5dr Leader 0 running 1} {pg2-xlln-b9ccfc4b8-cxbpn  0 running 1}]" func="util.ReplicationStatus()" file="util/failover.go:177" version=4.2.1
```

The reason for the missing name is the newly added regex in this commit: https://github.com/CrunchyData/postgres-operator/commit/323c374929420ccd9f1b9ce53d04fa3349e93164#diff-52f636d6fcde799d19d9392692712bc7R72

The randomly generated name of the replica set has a maximum of 10 digits but could be less according to the analysis here: https://stackoverflow.com/a/46298720

**Please tell us about your environment:**

* Deployment method: ansible
* Operating System: Ubuntu 18.04
* Where is this running: Private cloud
* Storage being used: Ceph
* Container Image Tag: 4.2.1
* PostgreSQL Version: 11.6
* Platform (Docker, Kubernetes, OpenShift): k8s 1.15

**Comment by jkatz** [comment]
@GeorgFleig Thanks for reporting and the analysis. I have proposed a patch that should fix the issue.

**Comment by jkatz** [comment]
@GeorgFleig Thanks for reporting. This will be available in `v4.2.2` in a few weeks.
